app/shared/user-agent-client.js

```diff
@@ -17,10 +17,15 @@ import { logger } from './logging';
 import { UserAgentHttpClient } from './user-agent-http-client';
 
 class UserAgentClient extends EventEmitter {
-  constructor() {
+  // You'd think { initialConnectDelayMillis = 50, maxConnectDelayMillis = 10000 } would work.  In
+  // fact, you'd be crying in your beer, just like me: it doesn't when you call simply `new
+  // UserAgentClient()`.
+  constructor(options) {
```

Comment:
Your comment directly addresses js syntax. It has nothing to do with with this being hard to debug, or renderer tests hanging. In fact, none of this is mentioned; as it stands, the comment is a rant about js syntax not being intuitive enough in your opinion. Such a comment has no value.